### ISSUE TITLE
Does this app allow the sending of SMS from the web interface?

### ISSUE THREAD

**Issue by Ruthalas** [nextcloud/ocsms#332]
(This is a question.)

Does this app allow the _sending_ of SMS from the web interface?

Neither the github readme nor the app page on Nextcloud clarify this.
(A single comment on the app page from three years ago says it did not.)

Thanks either way for your hard work- it's appreciated!

**Comment by Thovi98** [comment]
No, it’s not yet implemented. I would love to, but I am not able to make it possible. I know that @PhieF implemented it for /e/ (as you can see [here](https://github.com/nerzhul/ncsms-android/issues/230)). 

If someone has the skills to find the correct code to PR here, I would be grateful 😇
